fe/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -762,4 +762,13 @@ public void reset() {
         dataPartition = null;
         targetColumns.clear();
     }
+
+    @Override
+    public RedirectStatus getRedirectStatus() {
+        if (isExplain()) {
+            return null;
```

Comment:
Why not return `NO_FORWARD`?

Reply:
I referred  to here:
```
        if (redirectStatus == null) {
            return false;
        } else {
            return redirectStatus.isForwardToMaster();
        }
```
OK. I will change to `NO_FORWARD`.